services/ui-src/src/components/Login/Login.scss

```diff
@@ -7,9 +7,14 @@
   padding-bottom: 3em !important;
 }
 
-.login-wrapper form {
-  margin: auto;
-  input {
-    margin-left: 39%;
+.login-wrapper {
+  form {
+    label {
+      width: 100%;
+      text-align: left;
+    }
+    .form-control {
+      width: 100%;
+    }
```

Comment:
Thought: At 100% width, the text boxes are 8 pixels wider than the buttons above and below. Should we compensate here? `width: calc(100% - 0.5rem);`

Or is there a better way?

(Or does 8 pixels not matter compared to the 2 inch offset we had before?)

Reply:
checked with Ben on slack and this is good